network/router.go

```diff
@@ -166,6 +166,27 @@ func (r *Router) Send(e *ServerIdentity, msg Message) (uint64, error) {
 		return 0, errors.New("Can't send nil-packet")
 	}
 
+	// If sending to ourself, directly dispatch it
+	if e.Address == r.address {
```

Comment:
One question I have: is this check ok? Or should we compare the `NetworkAddress()`?

Reply:
Both addresses are strings, so this is OK. We could think about using the IP addresses, but they might not resolve.